### DISCUSSION TITLE
Turborepo with nestjs

### DISCUSSION THREAD

**Discussion by popuguytheparrot** [vercel/turborepo#4593] [Help]
Trying to connect turborepo + pnpm with nest. I create a packages/types package and try to plug it into apps/nest. I get an error "Error: Cannot find module '@ppg/types" What to do?

![image](https://user-images.githubusercontent.com/19804652/232315286-0b5a1485-0eba-43f3-aedc-89cb0c2d8585.png)

**Comment by anthonyshew** [comment]
We would need much more information to be able to help here. Could you provide a reproduction?
